spikesorters/tests/test_launcher.py

```diff
@@ -85,7 +85,7 @@ def test_run_sorters_dask():
     # create recording
     recording_dict = {}
     for i in range(8):
-        rec, _ = se.example_datasets.toy_example('toy_{}'.format(i), num_channels=8, duration=30, seed=0, dumpable=True)
```

Comment:
?!?!?!?!?!?!??!?!?!

Reply:
sad copy/paste
good that some firend can review my PR.